File: C++/Solution75.cpp
#include<iostream>
#include<vector>
#include<algorithm>
#include<unordered_map>
#include<unordered_set>
using namespace std;

class Solution75 {
public:
    void sortColors(vector<int>& nums) {
        int number[3];
        for(int i = 0; i < 3; i++){
            number[i] = 0;
        }
        for(auto ele: nums){
            number[ele]++;
        }
        int start = 0;
        for(int index = 0; index < 3; index++){
            for(int i = 0; i < number[index]; i++){
                nums[start] = index;
                start++;
            }
        }
    }
};

File: Java/other/Solution75.java
class Solution75 {
    public void sortColors(int[] nums) {
    	int[] freq = new int[3];
    	for(int ele: nums){
    		freq[ele] += 1;
    	}
    	for(int i = 0; i < freq[0]; i++){
    		nums[i] = 0;
    	}
    	for(int i = freq[0]; i < freq[0] + freq[1]; i++){
    		nums[i] = 1;
    	}
    	for(int i = freq[0] + freq[1]; i < nums.length; i++){
    		nums[i] = 2;
    	}
    }
}